packages/components/organisms/f-checkout/README.md

```diff
@@ -27,18 +27,24 @@
     npm install @justeat/f-checkout
     ```
 
-2.  Import the component
+2.  Import the component and its module, ensuring the latter is registered in the Vuex store upon creation.
```

Comment:
Not a change request – just wanted to know for my own knowledge. Why do we need to do this rather than it just being bundled?

Is it so that it becomes part of the global store?

If that's the case, should we look to namespace the store actions etc that are merged into the global store so we don't hit name collisions down the line?

Reply:
Before it was only registering the story module on mounted, however the component was being hidden with v-if / v-show, which meant it wasn't being mounted on form load. So pulling it out enables CoreWeb to register the state module, regardless of when the component is actually mounted